auto_round/autoround.py

```diff
@@ -997,7 +997,14 @@ def forward(m, hidden_states=None, *positional_inputs, **kwargs):
                             self.inputs[name][key] = [data]
                         else:
                             data = post_process_cache_data(self.batch_size, data, key)
-                            self.inputs[name][key] = list(torch.split(data, 1, dim=self.batch_dim))
+                            try:
+                                self.inputs[name][key] = list(torch.split(data, 1, dim=self.batch_dim))
+                            except:
+                                logger.error(
+                                f"this model has not been supported, "
+                                f"please raise an issue in https://github.com/intel/auto-round/issues"
```

Comment:
could batch_size to 1 automatically be set. we could throw an specific error and catch it in some function and try bs=1 again. if batch_size is already 1, just throw exception.